### DISCUSSION TITLE
periodic API fetch to get data and update a single field in table

### DISCUSSION THREAD

**Discussion by CodingRockz** [supabase/supabase#7993] [Questions]
I have an api link that returns a json data.

[{ id:1, name: "item1", price: 2000 }, {id: 2, name: "item2", price: 1500}...]

I have a table with a field called current_price. I want a periodic fetch of data from API (every 5 minutes for example) and map the items in my table by id and update the current_price to the price from API.

**Comment by GaryAustin1** [comment]
This has a way of doing that:
https://supabase.com/blog/2021/03/05/postgres-as-a-cron-server
